src/main/kotlin/ArcHiveOperations.kt

```diff
@@ -1,16 +1,18 @@
 import java.util.Scanner
 
-class ArcHiveOperations() {
-    private val scan = Scanner(System.`in`)
+class ArcHiveOperations {
+    val scan = Scanner(System.`in`)
     private val arcHives: MutableList<ArcHive> = mutableListOf()
 
 
     fun createArcHive() {
-        println("Введите название архива")
+        println("Введите название архива. Или \"-\" что бы вернуться.")
         val input = Scanner(System.`in`).nextLine()
         if (input.isEmpty()) {
             println("Вы не можете создать архив без заголовка")
             return
```

Comment:
🍏 Можно написать один return после всех условий, так как он должен выполниться при любом из условий.

```suggestion
if (input.isEmpty()) { 
...
 } else if (!input.equals("-")) {
 ...
 }
 
 return
```